src/components/AssetsPage/index.jsx

```diff
@@ -85,6 +86,8 @@ export default class AssetsPage extends React.Component {
     <React.Fragment>
       <div className={edxBootstrap.col}>
         { this.renderAssetsDropZone() }
+        <hr />
```

Comment:
This adds the line break. Right now, the break is gray. I know the mock-up had a black line. There's [this variable](https://github.com/edx/edx-bootstrap/blob/f0a6237174de7dd65815b07c73da127f082faa4d/sass/edx/_variables.scss#L276) commented out in edx-bootstrap. Should we uncomment it?

Reply:
There is a bootstrap class name that can add the line -- I believe it's the `page-header` class name https://getbootstrap.com/docs/3.3/components/#page-header

If that doesn't work I can dig in deeper.